_analyzers/supported-analyzers/phone-analyzers.md

```diff
@@ -0,0 +1,121 @@
+
+
+OpenSearch supports the following phone number analyzers:
+
+* `phone`: An [index analyzer]({{site.url}}{{site.baseurl}}/analyzers/index-analyzers/) to use at indexing time.
+* `phone-search`: A [search analyzer]({{site.url}}{{site.baseurl}}/analyzers/search-analyzers/) to use at search time.
+
+Internally, the plugin uses the [`libphonenumber`](https://github.com/google/libphonenumber) library and follows its parsing rules.
+
+The phone number analyzers are not meant to find phone numbers in larger texts. Instead, you should use them on fields which contain phone numbers alone.
+{: .note}
+
+## Installing the plugin
+
+Before you can use phone number analyzers, you must install the `analysis-phonenumber` plugin by running the following command:
+
+```sh
+./bin/opensearch-plugin install analysis-phonenumber
+```
+
+## Specifying a default region
+
+You can optionally specify a default region for parsing phone numbers by providing the `phone-region` parameter within the analyzer. Valid phone regions are ISO 3166 country codes. For more information, see [List of ISO 3166 country codes](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes).
+
+When tokenizing phone numbers containing the international calling prefix `+`, the default region is irrelevant. However, for phone numbers which either use a national prefix for international numbers (for example,  `001` instead of `+1` to dial Northern America from most European countries), the region needs to be provided. You can also properly index local phone numbers with no international prefix if you specify the region.
+
+## Example
+
+The following request creates an index containing one field, which ingests phone numbers for Switzerland (region code `CH`):
+
+```json
+PUT /example-phone
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "phone-ch": {
+          "type": "phone",
+          "phone-region": "CH"
+        },
+        "phone-search-ch": {
+          "type": "phone-search",
+          "phone-region": "CH"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "phoneNumber": {
+        "type": "text",
+        "analyzer": "phone-ch",
+        "search_analyzer": "phone-search-ch"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Analysing a (fictional) Swiss phone number with an international calling prefix will work the same with either the Swiss-specific phone region or without:
+```json
+GET /example-phone/_analyze
+{
+  "analyzer" : "phone-ch",
+  "text" : "+41 60 555 12 34"
+}
+```
+{% include copy-curl.html %}
+
+and
+
+```json
+GET /example-phone/_analyze
+{
+  "analyzer" : "phone",
+  "text" : "+41 60 555 12 34"
+}
+```
+{% include copy-curl.html %}
+
+will produce the same result:
+```json
+["+41 60 555 12 34", "6055512", "41605551", "416055512", "6055", "41605551234", ...]
```

Comment:
i used `...` here as i don't think there's much need in adding a ton of tokens to the example?

here's the complete output for the request above:
```json
{
    "tokens": [
        {
            "token": "+41 60 555 12 34",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 0
        },
        {
            "token": "6055512",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 1
        },
        {
            "token": "41605551",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 2
        },
        {
            "token": "416055512",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 3
        },
        {
            "token": "6055",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 4
        },
        {
            "token": "41605551234",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 5
        },
        {
            "token": "60555123",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 6
        },
        {
            "token": "41605",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 7
        },
        {
            "token": "4160",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 8
        },
        {
            "token": "4160555",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 9
        },
        {
            "token": "416055",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 10
        },
        {
            "token": "41 60 555 12 34",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 11
        },
        {
            "token": "60555",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 12
        },
        {
            "token": "605551",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 13
        },
        {
            "token": "6",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 14
        },
        {
            "token": "4160555123",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 15
        },
        {
            "token": "605551234",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 16
        },
        {
            "token": "416",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 17
        },
        {
            "token": "60",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 18
        },
        {
            "token": "605",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 19
        },
        {
            "token": "41",
            "start_offset": 0,
            "end_offset": 0,
            "type": "word",
            "position": 20
        }
    ]
}
```

Reply:
Yes, that's perfect. Thank you for adding the examples! I rewrote a bit, and I think we're ready to move this to editorial review, after which we can merge the PR.